Development-Guide/Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,47 @@
+== *Compiling and Flashing STM Board*
+
+----
+This has been tested on Linux Mint and is expected to work on other Debian based distros
+----
+
+
+=== Setup
+. Go to UCSolarCarTeam repo > Recruit Resources, and clone
+. Navigate to STM32F4Discovery-Resources and run the auto installer `sudo ./install.sh`. This will download STLink, gcc4mbed, and CubeMX2Makefile on your computer and add gcc4mbed to your path
```

Comment:
I'll change it so that it assumes they are added to the path, but I'll leave the manual adding in just in case they can't get it